app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx

```diff
@@ -122,7 +150,15 @@ export const FormAudience = ({
 
```

Comment:
It looks like we need this on Population % as well, according to Figma it is required.

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/13018240/101088385-3fa49e00-3579-11eb-80fc-b4ceb5b89240.png">

Reply:
Ana is out sick but maybe @jaredlockhart will know, is Population % a required field? If so it is missing from the GQL check:

![Screen Shot 2020-12-03 at 4 38 23 PM](https://user-images.githubusercontent.com/6392049/101091422-26561e80-3586-11eb-87ce-d61d5fcc4ca6.png)

If this is indeed required I'll file an issue.

Reply:
Ah interesting. It's definitely possible it's not actually a required field and Figma is wrong, we'll defer to Jared :P

Reply:
Jared confirmed this is required.

Two follow up issues have been created for this: #4116 + #4117